client/src/components/OnboardUser.jsx

```diff
@@ -95,6 +95,10 @@ const OnboardUser = props => {
       [e.target.name]: e.target.value,
     });
   };
+  async function onSubmit() {
+    await setDemographicsComorbiditiesThunk(formState, userSession);
```

Comment:
```suggestion
    setDemographicsComorbiditiesThunk(formState, userSession);
```

No need to `await` here

Reply:
You're right! Just fixed it and will push, thanks!